rtt/transports/mqueue/MQSendRecv.cpp

```diff
@@ -75,11 +76,14 @@ void MQSendRecv::setupStream(base::DataSourceBase::shared_ptr ds, base::PortInte
     marshaller_cookie = mtransport.createCookie();
     mis_sender = is_sender;
 
-    std::stringstream namestr;
-    namestr << '/' << port->getInterface()->getOwner()->getName() << '.' << port->getName() << '.' << this << '@' << getpid();
-
     if (policy.name_id.empty())
-        policy.name_id = namestr.str();
+    {
+        std::stringstream name_stream;
+        name_stream << port->getInterface()->getOwner()->getName() << '.' << port->getName() << '.' << this << '@' << getpid();
```

Comment:
OK ... Since you leave it to me, I'll throw.

For your information "global" means "system wide" in this context.